packages/babel-core/register.js

```diff
@@ -1,3 +0,0 @@
-/* eslint max-len: 0 */
-// TODO: eventually deprecate this console.trace("use the `babel-register` package instead of `babel-core/register`");
-module.exports = require("babel-register");
```

Comment:
This was deprecated for a long time, so I think we should remove it. Not sure if we should do a warning.